src/routes/JobOpportunity/JobOpportunity.tsx

```diff
@@ -1,9 +1,217 @@
+        <ActionButton
+          label={t('compare')}
+          icon={<MdCompareArrows size={24} className="text-accent" />}
+          onClick={notImplemented}
+          className="bg-todo"
+        />
+        <ActionButton
+          label={t('create-path')}
+          icon={<MdOutlineRoute size={24} className="text-accent transform rotate-90 -scale-x-100" />}
+          onClick={notImplemented}
+          className="bg-todo"
+        />
+        <ActionButton
+          label={t('share')}
+          icon={<MdOutlineShare size={24} className="text-accent" />}
+          onClick={notImplemented}
+          className="bg-todo"
+        />
+        <ActionButton
+          label={t('print')}
+          icon={<MdOutlinePrint size={24} className="text-accent" />}
+          onClick={window.print}
+        />
+      </div>
+      <div className="flex flex-col gap-7">
+        <div>
+          <h2 id={t('job-opportunity.description')} className="text-heading-2 scroll-mt-[96px]">
+            {t('job-opportunity.description')}
+          </h2>
+          <p className="text-body-md font-arial mt-5">{getLocalizedText(tyomahdollisuus?.kuvaus)}</p>
+        </div>
+        <div>
+          <h2 id={t('job-opportunity.most-common-job-tasks.title')} className="text-heading-2 scroll-mt-[96px]">
+            {t('job-opportunity.most-common-job-tasks.title')}
+          </h2>
+          <p className="text-body-md font-arial my-5">{t('job-opportunity.most-common-job-tasks.description')}</p>
+          <ol className="list-decimal ml-7 text-body-lg font-medium text-black leading-7">
+            {ammatit.map((ammatti) => (
+              <li
+                className="text-capitalize text-body"
+                title={`${ammatti.koodi} ${getLocalizedText(ammatti.kuvaus)} (${ammatti.osuus.toFixed(1)}%, N = ${clusterSize})`}
+                key={ammatti.uri}
+              >
+                {getLocalizedText(ammatti.nimi)}
+              </li>
+            ))}
+          </ol>
+        </div>
+        <div>
+          <h2 id={t('job-opportunity.key-figures.title')} className="text-heading-2 scroll-mt-[96px]">
+            {t('job-opportunity.key-figures.title')}
+          </h2>
+          <p className="text-body-md font-arial mb-6 mt-4">{t('job-opportunity.key-figures.description')}</p>
+          <div className="bg-todo h-[380px]" />
+        </div>
+        <div>
+          <h2 id={t('job-opportunity.labour-market-picture.title')} className="text-heading-2 scroll-mt-[96px]">
+            {t('job-opportunity.labour-market-picture.title')}
+          </h2>
+          <p className="text-body-md font-arial mb-6 mt-4">{t('job-opportunity.labour-market-picture.description')}</p>
+          <div className="bg-todo h-[380px]" />
+        </div>
+        <div>
+          <h2 id={t('job-opportunity.salary-trends.title')} className="text-heading-2 scroll-mt-[96px]">
+            {t('job-opportunity.salary-trends.title')}
+          </h2>
```

Comment:
It came as an idea to have a component for this as this same pattern repeats so many time.

```tsx
const H2Scroll = ({ title }: { title: string }) => {
  return (
    <h2 id={title} className="text-heading-2 scroll-mt-[96px]">
      {title}
    </h2>
  );
};
```

Usage:
```tsx
<H2Scroll title={t('job-opportunity.key-figures.title')} />
```

Reply:
The heading level could be a prop. This way, it would be easier to use in different situations.